examples/local-backup/README.md

```diff
@@ -0,0 +1,174 @@
+# Local Backup with Velero and Minio
+
+This example creates a configuration that allows you to back up Kubernetes objects
+to your laptop (or wherever you are running idpbuilder from).
+
+In short, it:
+1. Creates a [MinIO](https://min.io/) installation that mounts a local directory.
+2. Creates a [Velero](https://velero.io/) installation that targets the in-cluster MinIO storage.
+
+## Installation
+
+First, we need to ensure the local cluster is configured to mount a local directory.
+This is done through the kind configuration file that you can supply to `idpbuilder`.
+
+Take a look at the [kind.yaml](./kind.yaml) file. The most relevant part is this bit:
+
+```yaml
+nodes:
+  - role: control-plane
+    extraMounts:
+      - hostPath: /home/ubuntu/backup # replace with your own path
+        containerPath: /backup
+```
+
+This instructs Kind to make your machine's directory at `/home/ubuntu/backup`
+available at `/backup` for the Kubernetes node.
+
+You **must** change this value for your own setup. This directory also must exist on your machine.
+For example, you may want to change it to `/Users/my-name/backup`.
+
+Once you've made the change, run this command from the root of this repository.
+
+```bash
+# example: mkdir /Users/my-name/backup
+mkdir <path/to/directory> 
+
+idpbuilder create --kind-config examples/local-backup/kind.yaml --kind-config examples/local-backup/kind.yaml
+```
+
+This command:
+1. Creates a standard idpbuilder installation, a kind cluster and core packages (ArgoCD, Gitea, and Ingress-Nginx).
+2. Creates two custom packages: [MinIO](./minio.yaml) and [Velero](./velero.yaml).
+
+Once the command exits, you can check the status of installation by going to https://argocd.cnoe.localtest.me:8443/applications.
+
+Username is `admin`, and password is obtained with:
+```bash
+kubectl -n argocd get secret argocd-initial-admin-secret -o go-template='{{ range $key, $value := .data }}{{ printf "%s: %s\n" $key ($value | base64decode) }}{{ end }}'
+```
```

Comment:
This bit is not required. it gets printed every time someone runs the idpBuilder